tests/test_slim_engine.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for SLiM simulation engine.
+"""
+import sys
+import unittest
+import tempfile
+
+import tskit
+
+import stdpopsim
+import stdpopsim.cli
+from . test_cli import capture_output
+
```

Comment:
Probably simplest to skip the entire module if we're on Windows. I think unittest will allow that. If not, we can skip the classes.

Reply:
I kind of expected some tests to fail on windows because `_SLiMEngine.get_version()` calls out to the slim binary. Have you disabled the travisCI windows build? I didn't manage to find the logs for that.

In princial, it should be possible to generate a slim script without having a slim binary available. I think this should work fine, if get_version() is fixed to return `unknown` or something.

Reply:
I think we should fail early on Windows, and let the user know it's just not supported because SLiM doesn't build on Windows (yet). So, there's no point in finessing the tests here IMO.

It's only Appveyor is on Windows. TravisCI should be doing OSX and LInux, but we haven't pulled in the conda package yet AFAIK

Reply:
See #397 for discussion on conda package.

Reply:
Ok. To be honest, I'm a little confused with all the different CI stuff. Is there something I need to include in the test_slim_engine.py file for the skipping or erroring out that you're suggesting? (Eg. test for windows at the top of the file and raise X). Or does this go somewhere else, like in the appveyor.yml?

test_engines.py currently calls get_engine() for all registered engines. So this test will need to be changed, or the get_version() should be amended to return something sensible if the slim binary can't be found.

Reply:
Yes, it's a bit confusing all right. Might be easier to have a chat about this: you free tomorrow at any point?